test/topology_custom/test_group0_schema_versioning.py

```diff
@@ -124,7 +124,8 @@ async def test_schema_versioning_with_recovery(manager: ManagerClient):
     Verify that schema versions are in sync after each schema change.
     """
     cfg = {'enable_user_defined_functions': False,
-           'force_gossip_topology_changes': True}
+           'force_gossip_topology_changes': True,
+           'enable_tablets': False}
```

Comment:
This should be implicit from `force_gossip_topology_changes`. What happens when you skip it? Is there an exception?

Reply:
At the time this pull request was submitted htere was no check., so one would get runtime errors. It was addressed separately by @bhalevy in his patch for https://github.com/scylladb/scylladb/issues/21075